### PR TITLE
python fixes

### PR DESCRIPTION
**Pull request by farizrahman4u**

### CODE REVIEW

**Comment by maxpumperla** [comment]
@farizrahman4u what does it fix?

**Comment by farizrahman4u** [comment]
@maxpumperla test added

File: konduit-serving-test/src/test/java/ai/konduit/serving/util/python/PythonVariablesTest.java [hunk 1]
```diff
@@ -76,7 +86,17 @@ private void testInsertGet(PythonVariables pythonVariables,String key,Object val
         assertNull(pythonVariables.getValue(key));
         pythonVariables.setValue(key,value);
         assertNotNull(pythonVariables.getValue(key));
-        assertEquals(expectedValue,pythonVariables.getValue(key));
+        Object actualValue = pythonVariables.getValue(key);
+        if (expectedValue instanceof Object[]){
+            assert actualValue instanceof Object[];
+            Object[] actualArr = (Object[])actualValue;
```
Comment (agibsonccc) [inline konduit-serving-test/src/test/java/ai/konduit/serving/util/python/PythonVariablesTest.java]: Use assertTrue not the java assert keyword

**Comment by maxpumperla** [comment]
@farizrahman4u thanks!

**Pull request merged**